src/runtime/slot-consumer.ts

```diff
@@ -81,7 +82,7 @@ export class SlotConsumer {
     return [...this.generateProvidedContexts()];
   }
 
-  findProvidedContext(predicate: (_: ProvidedSlotContext) => boolean) {
+  findProvidedContext(predicate: Predicate<ProvidedSlotContext>) {
```

Comment:
did you skip the one on line 89 on purpose?

Reply:
Thanks!  I looked for the regex `=>.boolean`, but this was a specific usage. (=> true)

Fixed.